backfill.sql

```diff
@@ -97,14 +101,16 @@ $$ LANGUAGE PLPGSQL VOLATILE;
 -- The main event
 -- staging_table is the (possibly temporary) table from which rows will be moved 
 -- destination_hypertable is the table where rows will be moved
--- on_conflict_do_nothing controls whether duplicate rows are ignored
+-- on_conflict_action controls whether duplicate rows are being ignored (DoNothing, default), an update clause is generated based on on_conflict_columns (Update), or if the duplicate row raises an error (Error) 
 -- delete_from_staging specifies whether we should delete from the staging table as we go or leave rows there
 -- compression_job_push_interval specifies how long push out the compression job as we are running, ie the max amount of time you expect the backfill to take
+-- update_action_columns is an array of columns to use in the update clause when a conflict arises and the on_conflict_action is set to 'Update'
```

Comment:
I would probably name this: `on_conflict_update_columns`

Reply:
I'd also add a quick error checking step to check that this is set IFF on_conflict_do='UPDATE' and throw an error saying so.